src/ApplicationManager.cpp

```diff
@@ -150,6 +150,15 @@ void ApplicationManager::runOneLoopCycle() {
 void ApplicationManager::decidePlay(world::World *_world) {
     playChecker.update(_world);
 
+    // Here for manual change with the interface
+    if(rtt::ai::stp::PlayDecider::change) {
+        auto validPlays = playChecker.getValidPlays();
+        currentPlay = playDecider.decideBestPlay(_world, validPlays);
+        currentPlay->updateWorld(_world);
+        currentPlay->initialize();
+        rtt::ai::stp::PlayDecider::change = false;
```

Comment:
Wouldn't this make it so that the net iteration it'll switch back to whatever play it wants to?